src/MultiOwnable.sol

```diff
@@ -137,8 +136,7 @@ contract MultiOwnable {
     /// @param index The index of the owner to be removed.
     /// @param owner The ABI encoded bytes of the owner to be removed.
     function removeLastOwner(uint256 index, bytes calldata owner) external virtual onlyOwner {
-        MultiOwnableStorage storage $ = _getMultiOwnableStorage();
-        uint256 ownersRemaining = $.nextOwnerIndex - $.removedOwnersCount;
+        uint256 ownersRemaining = ownerCount();
```

Comment:
I think we can get rid of this stack variable and do the same thing as in `removeOwnerAtIndex():

```solidity
if (ownerCount() > 1) {
    revert NotLastOwner(ownersRemaining);
}
```

Reply:
but then I would need to call `ownerCount` again here `revert NotLastOwner(ownersRemaining);`

Reply:
Would be `revert NotLastOwner(ownerCount());`

Reply:
Oh yes right, I missed that.

Reply:
But yeah could debate: do we penalize gas on the non revert path in order to save on revert?